.github/workflows/dockerhub.yml

```diff
@@ -17,6 +17,10 @@ on:
       DEPLOYMENT_DOCKER_USER_KEY:
         required: true
         description: "The access key for the user defined by DEPLOYMENT_DOCKER_USER"
+    outputs:
+      docker_tag:
+        description: "The tag (version number) of the image that was published to Docker Hub."
+        value: ${{ jobs.DockerHub.outputs.docker_tag }}
```

Comment:
This adds a `docker_tag` output to the workflow, where it can be accessed by the caller workflow. The value of the `docker_tag` output is that of the similarly-named `docker_tag` output from the `DockerHub` job below